src/main/java/com/cleanroommc/groovyscript/compat/mods/thaumcraft/aspect/AspectHelper.java

```diff
@@ -0,0 +1,68 @@
+package com.cleanroommc.groovyscript.compat.mods.thaumcraft.aspect;
+
+import com.cleanroommc.groovyscript.api.GroovyLog;
+import net.minecraft.item.ItemStack;
+import net.minecraftforge.common.MinecraftForge;
+import thaumcraft.api.ThaumcraftApi;
+import thaumcraft.api.aspects.AspectEventProxy;
+import thaumcraft.api.aspects.AspectList;
+import thaumcraft.api.aspects.AspectRegistryEvent;
+
+public class AspectHelper {
+
+    private String entity;
+    private ItemStack object;
+    private AspectList aspects = new AspectList();
+
+    public AspectHelper() {
+        //do nothing
+    }
+
+    public AspectHelper entity(String entity) {
+        this.entity = entity;
+        return this;
+    }
+
+    public AspectHelper object(ItemStack object) {
```

Comment:
`object` only works on `ItemStack`s, despite `registerObjectTag` working on both `ItemStack`s and `oreDict`s (with oredict as a string)

Reply:
fixed in commit: [7111713](https://github.com/CleanroomMC/GroovyScript/pull/18/commits/711171376859d15442bb8918f55e227b82876c76)